HW_2/SQLQuery2.sql

```diff
@@ -0,0 +1,83 @@
+/****** Script for SelectTopNRows command from SSMS  ******/
```

Comment:
Please fix the file encoding to recognize Cyrillic characters. See last comment with screenshot https://stackoverflow.com/questions/34005918/sql-encoding-does-not-recognise-cyrillic